packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -43,6 +43,7 @@ function LoginForm({
     const form = formToJson<LoginFormParams>(e.currentTarget);
     onPasswordLogin(form);
   };
+  const [hasFocused, setHasFocused] = useState(autoFocus);
```

Comment:
Technically the naming is in reverse here, might cause confusion down the line.
You set hasfocus to false, but actually you focussed it.

Maybe shouldFocus is better in that case?
Or negate the start to be reversed !autoFocus and set it to true